nlbq/api.py

```diff
@@ -11,7 +10,7 @@
 
 
 @app.get("/api/dry_run")
-async def dry_run(q: Union[str, None] = None):
+async def dry_run(q: str | None = None):
```

Comment:
`q` is required, so this can be simply 

```suggestion
async def dry_run(q: str):
```

(I realise I need to fix this in `nldb` too)

Reply:
https://github.com/tomdyson/nldb/pull/7

I should update that to require python 3.11+ though

Reply:
[d255190](https://github.com/torchbox/nlbq/pull/6/commits/d255190599efe828c1ef83b694978139c8554845)